src/player.cpp

```diff
@@ -8528,12 +8528,19 @@ hint_rating player::rate_action_reload( const item &it ) const
 
 hint_rating player::rate_action_unload( const item &it ) const
 {
-    if( ( it.is_container() || it.is_gun() || it.is_bandolier() ) && !it.contents.empty() ) {
-        // gunmods can also be unloaded
+    if( ( it.is_container() || it.is_bandolier() ) && !it.contents.empty() ) {
```

Comment:
* Attach Masterkey shotgun to Remington 700
* Reload the modded gun with 00 shot
* Try to unload
* No option to do so

Should probably go through all gunmods and check if those can be reloaded.

Reply:
Fixed.